package.json

```diff
@@ -29,17 +31,56 @@
     "url": "https://github.com/education/classroom-desktop/issues"
   },
   "homepage": "https://github.com/education/classroom-desktop#readme",
+  "config": {
+    "forge": {
```

Comment:
Any changes related to tweaking the installers that `electron-forge` creates should be able to be done in here